plugins/modules/packaging/os/xbps.py

```diff
@@ -50,21 +50,41 @@
             - Whether or not to upgrade whole system
         type: bool
         default: 'no'
+    upgrade_xbps:
+        description:
+            - Whether or not to upgrade the xbps package when necessary.
+              Before installing new packages,
+              xbps requires the user to update the xbps package itself.
+              Thus when this option is set to C(no),
+              upgrades and installations will fail when xbps is not up to date.
+        type: bool
+        default: 'yes'
 '''
 
 EXAMPLES = '''
-# Install package foo
-- xbps: name=foo state=present
-# Upgrade package foo
-- xbps: name=foo state=latest update_cache=yes
-# Remove packages foo and bar
-- xbps: name=foo,bar state=absent
-# Recursively remove package foo
-- xbps: name=foo state=absent recurse=yes
-# Update package cache
-- xbps: update_cache=yes
-# Upgrade packages
-- xbps: upgrade=yes
+- name: Install package foo (automatically updating the xbps package if needed)
+  xbps: name=foo state=present
```

Comment:
Doesn't need doing now, though for reference, this format is better for Playbooks:

```suggestion
- name: Install package foo (automatically updating the xbps package if needed)
  xbps:
    name: foo
    state: present
```